openprovider/apis/params_builder/params_creator.php

```diff
@@ -25,6 +25,19 @@ public function createParameters(array $args, $client, string $method): array
             $this->createParametersGetDelete($args, $client, $method);
     }
 
+    /**
+     * @param string $name
+     * @return string converted from idn to ascii
+     */
+    protected function idnEncode(string $name): string
```

Comment:
It's better to move it to some helper because you can need to use it not only inside the params creators.